controllers/mirrorpeer_controller.go

```diff
@@ -241,45 +247,238 @@ func (r *MirrorPeerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, err
 	}
 
+	if utils.IsStorageClientType(mirrorPeer.Spec.Items) {
+		result, err := createStorageClusterPeer(ctx, r.Client, logger, mirrorPeer)
+		if err != nil {
+			logger.Error("Failed to create StorageClusterPeer", "error", err)
+			return result, err
+		}
+	}
 	return r.updateMirrorPeerStatus(ctx, mirrorPeer)
 }
 
+func getKey(clusterName, clientName string) string {
+	return fmt.Sprintf(ClientConfigMapKeyTemplate, clusterName, clientName)
+}
```

Comment:
Do we really need this? I think Sprintf is good enough.

Reply:
I prefer it this way as it is more verbally descriptive. I will remove the clientconfigmapkey and use a format string